Yoast/wordpress-seo
# Add test for the remove_replacement_variable action

## Summary

This PR can be summarized in the following changelog entry:

* Adds a test for the `remove_replacement_variable` action

## Test instructions

This PR can be tested by following these steps:

* Run `yarn test` and see if all tests pass

## Quality assurance


Fixes #9746
